src/mozmlops/templates/template_flow.py

```diff
@@ -0,0 +1,111 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import os
+
+from metaflow import (
+    FlowSpec, IncludeFile, Parameter,
+    card, current, step,
+    environment, kubernetes
+)
+from metaflow.cards import Markdown
+
+from artifact_store import ArtifactStore
```

Comment:
The import is no longer in this directory, so this statement will make the flow fail

Reply:
Good catch. I haven't tried to run these flows from here yet. I'll do that today.

Reply:
OK. Fixed. Ran the `show` command to confirm that the change works!

Reply:
I believe the fix was mistakenly committed to #4 . I'm fine with merging this in the broken state, and take further action in #4